Tools/CISupport/ews-build/steps.py

```diff
@@ -1860,10 +1860,19 @@ def run(self):
         remote = self.getProperty('remote', DEFAULT_REMOTE)
         validators = self.VALIDATORS_FOR.get(remote, [])
         lower_case_reviewers = [reviewer.lower() for reviewer in reviewers]
-        if validators and not any([validator.lower() in lower_case_reviewers for validator in validators]):
+        lower_case_validators = [validator.lower() for validator in validators]
```

Comment:
This seems confusing, previous code was better in terms of readability, and seems like it did the same thing.

Another alternative is to rename lower_case_validators to validators (store in same variable name after converting to lower-case)